docs/source/tutorials/token_classification/entity_recognition_training.ipynb

```diff
@@ -0,0 +1,737 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "59db66ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture \n",
+    "!pip install transformers tqdm "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e20de927",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# packages version: \n",
+    "# numpy == 1.23.0 \n",
+    "# tqdm == 4.64.0 \n",
+    "# transformers == 4.20.1 "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0d2e007",
+   "metadata": {},
+   "source": [
+    "# Token Classification Label Error Detection - Part 1 "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a6676421",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we show how you can retrieve the model-predictive probabilities and labels from a NLP token-classification dataset. These inputs are used to identify potential label issues in the dataset. \n",
```

Comment:
Add placeholder link here for linking to the token_classification.ipynb tutorial

Reply:
Should mention early on in notebook: the specific token classification task we consider here is Named Entity Recognition with the CoNLL-2003 dataset.